src/ledger/LedgerManagerImpl.h

```diff
@@ -42,58 +42,64 @@ class BasicWork;
 class LedgerManagerImpl : public LedgerManager
 {
   protected:
-    struct CloseLedgerOutput
+    Application& mApp;
+    std::unique_ptr<XDROutputFileStream> mMetaStream;
+    std::unique_ptr<XDROutputFileStream> mMetaDebugStream;
+    std::weak_ptr<BasicWork> mFlushAndRotateMetaDebugWork;
+    std::filesystem::path mMetaDebugPath;
+
+  private:
+    // Output of the apply process, also what gets held as "LCL".
+    struct LedgerState
```

Comment:
Good idea, I like the inner struct breakdown in the interim of hopefully making this class less icky.